src/DevBetterWeb.Infrastructure/Data/AppDbContext.cs

```diff
@@ -34,6 +36,31 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         public override async Task<int> SaveChangesAsync(CancellationToken cancellationToken = new CancellationToken())
         {
+            if (_dispatcher is not null)
```

Comment:
Why are we adding this code here when there is already code to handle every kind of domain event after the call to `base.SaveChangesAsync()` ?